static/css/components/Share.less

```diff
@@ -0,0 +1,158 @@
+@import (less) "less/colors.less";
```

Comment:
Any styles for share links should be added to the existing `shareLinks.less` file, [here](https://github.com/internetarchive/openlibrary/blob/master/static/css/components/shareLinks.less).  I suspect that much of this new CSS will not be needed once you switch to Colorbox.